### DISCUSSION TITLE
Virtual bioequivalence

### DISCUSSION THREAD

**Discussion by ani199797** [Open-Systems-Pharmacology/Forum#1518] [Questions & Problems]
1. Is it possible to do virtual bioequivalence in PK Sim?
2. how can we extract the individual subject PK data from the population PK graph?
3. is there a way to get the 90% confidence interval in PK Sim?

**Comment by StephanSchaller** [comment]
1. Yes
2. RIght-click on your simulation (simulation has to be run before): 
![image](https://user-images.githubusercontent.com/25477643/235307574-45b9e0a0-15b7-45be-a061-ec8239b22316.png)
3.  Use the 90 and 10 %tile outputs: 
![image](https://user-images.githubusercontent.com/25477643/235307621-2a0f6c0d-4536-4fa9-bcb6-329c77b3563f.png)

**Reply by ani199797** [reply]
Thank you

**Reply by msevestre** [reply]
Exporting to CSV always export to internal units at the moment.
If you want to convert to custom unit, export to excel (if it's available, I can't remember)

**Reply by ani199797** [reply]
Thank you.
